docs/version-history/feature-versions-24-x.rst

```diff
@@ -15,8 +15,16 @@ Release Notes:
 
 - This version includes all the updates from :ref:`lts-version-history-2403`.
 
-New Features:
+- Methods in :class:`htcondor2.Schedd` which take ``job_spec`` arguments now
+  accept :class:`int` cluster IDs.  These functions
```

Comment:
```suggestion
  accept a cluster ID in the form of an :class:`int`.  These functions
```
Just a suggestion because "accept `int` cluster IDs" seems clunky to me.